brand/models/brand.py

```diff
@@ -195,21 +189,6 @@ def refresh(self, name=True, description=True, countries=True, overwrite_existin
         if countries:
             self.refresh_countries()
 
-    def subclass(self):
```

Comment:
This should and the related tests should be migrated to datasource.py. It's more useful than you might expect.

Reply:
done; created a test of it in BanktrackTestCase